public/docs/_examples/animations/ts/src/hero-list-basic.component.ts

```diff
@@ -0,0 +1,56 @@
+ // #docregion
+ // #docregion imports
+import {
+  Component,
+  Input,
+  animation,
+  state,
+  style,
+  transition,
+  animate
+} from '@angular/core';
+// #enddocregion imports
+import {Hero, Heroes} from './hero.service';
+
+@Component({
+  moduleId: module.id,
+  selector: 'hero-list-basic',
+  // #docregion template
+  template: `
+    <div *ngFor="let hero of heroes.get()"
+         @heroState="hero.state"
+         (click)="hero.toggleActivation()">
+      {{hero.name}}
+    </div>
+  `,
+  // #enddocregion template
+  styleUrls: ['hero-list.component.css'],
+  // #docregion animationdef
+  animations: [
+    animation('heroState', [
+      // Styles in each state
+      state('void',     style({})),
+      state('inactive', style({
+        backgroundColor: '#eee',
+        transform: 'scale(1)'
+      })),
+      state('active',   style({
+        backgroundColor: 'lightblue',
+        transform: 'scale(1.1)'
+      })),
+      // Transitions between states
+      transition('inactive => active', [
+        style(':inactive'),
+        animate('100ms ease-in', style(':active'))
+      ]),
+      transition('active => inactive', [
+        style(':active'),
```

Comment:
This gets applied automatically now. There is no need to reference the starting state.